e2e/resources/workbench-config.ts

```diff
@@ -31,6 +31,7 @@ const local: IEnvConfig = {
   ADMIN_TEST_USER: process.env.ADMIN_TEST_USER || 'puppeteer-admin-1@fake-research-aou.org',
   DEFAULT_CDR_VERSION_NAME: 'Synthetic Dataset v3',
   ALTERNATIVE_CDR_VERSION_NAME: 'Synthetic Dataset v3 with WGS',
+  CONTROLLED_TIER_CDR_VERSION_NAME: 'Synthetic Dataset in the Controlled Tier',
```

Comment:
this doesn't have WGS right now, so I would expect nightly to fail in test without https://precisionmedicineinitiative.atlassian.net/browse/RW-7207

Reply:
Yes, nightly tests fail. right now, the PR is blocked by RW-7207.